lib/linkerd2-request-filter/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "linkerd2-request-filter"
```

Comment:
AFAICT, the difference between this and [`tower-filter`](https://github.com/tower-rs/tower/blob/master/tower-filter/src/lib.rs) is that the predicate here is synchronous rather than a future?

It seems like eventually, this would be good to upstream into `tower-filter`. Not a blocker for now.